packages/terra-application-name/src/ApplicationMenuName.scss

```diff
@@ -5,17 +5,24 @@
     color: var(--terra-application-menu-name-color, #000);
     display: flex;
     flex: var(--terra-application-menu-name-flex, 0 0 auto);
+    max-width: 100%;
+    padding-bottom: var(--terra-application-menu-name-padding-bottom, 0.3571rem);
+    padding-top: var(--terra-application-menu-name-padding-top, 0.3571rem);
+    position: relative;
   }
 
   .accessory {
+    flex: 0 0 auto;
+    max-width: 100%;
     padding-left: var(--terra-application-menu-name-accessory-padding-left, 0.3571rem);
   }
 
   .title {
+    flex: 0 1 auto;
     font-weight: var(--terra-application-header-name-title-font-weight, normal);
```

Comment:
Oops - for consistency, we should set the default font-weight here to bold to match the header-name, or set both to normal.